src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclint/Checker.java

```diff
@@ -195,6 +192,18 @@ public Void scan(DocCommentTree tree, TreePath p) {
                     reportMissing("dc.missing.comment");
                 }
                 return null;
+            } else if (tree.getFirstSentence().isEmpty() && !isOverridingMethod) {
+                if (tree.getBlockTags().isEmpty()) {
+                    reportMissing("dc.empty.comment");
+                    return null;
+                } else {
+                    DocTree firstTag = tree.getBlockTags().get(0);
+                    // Don't report an empty description if the comment begins with @deprecated,
+                    // because javadoc will use the content of that tag in summary tables.
+                    if (firstTag.getKind() != DocTree.Kind.DEPRECATED) {
```

Comment:
Various reasons, 
1. It seems the convention is to simply prefix an existing comment with `@deprecated` or to replace the existing body description with `@deprecated reason-why-deprecated`
2. This is only for the case when there is no body description; it seems hard to imagine that someone would start a comment with `@see` `@param` `@return` etc and then have the `@deprecated` tag.

That being said, it would be easy enough to change the code to check for any instance of `@deprecated`.

Reply:
Given that the downstream code does not impose any ordering restrictions on the tags, it is probably with doing the same here.